src/components/WithdrawMilestoneFundsButton.jsx

```diff
@@ -32,11 +33,12 @@ class WithdrawMilestoneFundsButton extends Component {
           title: isRecipient ? 'Withdrawal Funds to Wallet' : 'Disburse Funds to Recipient',
           content: React.swal.msg(
             <div>
-              {donationsCount > 5 && (
+              {donationsCount > config.donationCollectCountLimit && (
```

Comment:
This might need to be replaced with ">=" so the config.donationCollectCountLimit matches the text below

Reply:
If donatoinsCount === config.donationCollectCountLimit , all donations can be collected in one transaction. So `>` seems to be correct.